pkgs/desktops/gnome-3/3.24/core/gnome-system-monitor/default.nix

```diff
@@ -10,16 +10,19 @@ stdenv.mkDerivation rec {
   propagatedUserEnvPkgs = [ gnome3.gnome_themes_standard ];
 
   buildInputs = [ bash pkgconfig gtk3 glib intltool itstool libxml2
-                  gtkmm3 libgtop wrapGAppsHook
+                  gtkmm3 libgtop makeWrapper
                   gdk_pixbuf gnome3.defaultIconTheme librsvg
-                  gnome3.gsettings_desktop_schemas ];
+                  gnome3.gsettings_desktop_schemas systemd ];
 
   preFixup = ''
-    gappsWrapperArgs+=(
-      --prefix XDG_DATA_DIRS : "${gtk3.out}/share:${gnome3.gnome_themes_standard}/share"
-    )
+    wrapProgram "$out/bin/gnome-system-monitor" \
```

Comment:
Rebasing someone else's branch against master on top of staging is quite confusing :sweat: